fvm_actor_utils/src/messaging.rs

```diff
@@ -152,7 +152,7 @@ impl Messaging for FvmMessenger {
 ///
 #[derive(Debug)]
 pub struct FakeMessenger {
-    pub last_message: RefCell<Option<RawBytes>>,
+    pub last_message: RefCell<Option<Option<IpldBlock>>>,
```

Comment:
Do we really need `Option<Option<>>`? This at least needs documentation to explain it.